pom.xml

```diff
@@ -77,15 +68,15 @@
                         </goals>
                         <configuration>
                             <from>
-                                <image>amazoncorretto:21</image>
+                                <image>amazoncorretto:17</image>
                             </from>
                             <container>
-                                <mainClass>io.numaproj.pulsar.PulsarApplication</mainClass>
+                                <mainClass>com.numaproj.pulsar.SpringPulsarApplication</mainClass>
```

Comment:
Please continue using `PulsarApplication`. Spring is the way we implement Pulsar, normally when we are naming, we use WHAT and exclude HOW.